### DISCUSSION TITLE
[HxPager] Can you increase the width on both desktop and mobile views?

### DISCUSSION THREAD

**Discussion by bholbrook** [havit/Havit.Blazor#628] [Q&A]
Is there some way to increase the width the HxPager takes up on the screen?

I'm finding that by default in a desktop browser on a large monitor the pager spans wide enough to fill ten pages of data, but on mobile it only shows page one but the component itself only takes up about half the horizontal width of the page.

Is there any way to change this please?

Desktop resolution:
![image](https://github.com/havit/Havit.Blazor/assets/763124/0f7b8110-0b07-4f59-9db8-40eef9779501)

Mobile resolution:
![image](https://github.com/havit/Havit.Blazor/assets/763124/aeb27448-96d9-439e-a7bf-d06b6f452db4)

**Comment by hakenr** [comment]
Currently not, the responsive behavior is hardcoded. Feel free to submit a PR if you feel this should be improved.
